src/v1/notifications-unsubscribe.ts

```diff
@@ -0,0 +1,63 @@
+import { Client } from '../client'
+import { BaseError } from '../errors'
+import { UriHelper } from '../uri-helper'
+import { ThBaseHandler } from '../base'
+
+export interface NotificationsUnsubscribeOptions {
+  base?: string
+  uri?: string
+  user: undefined
+}
+
+export interface NotificationsUnsubscribeResponse {
+  msg: string
+}
+
+export class NotificationsUnsubscribe extends ThBaseHandler {
+  public static baseEndpoint = '/api/v1/notifications/unsubscribe'
```

Comment:
I'm doing this because this because it should work without authorization.
That means we shouldn't take `tenantId` from user, we should accept it from method's arguments, which brings problem with `generateBaseUri`, because it's trying to take tenantId from user.
That's why I'm making a separate controller and passing `user: undefined` to it.
Adding normal method there will make them unusable